src/main/java/com/google/error_analyzer/backend/LogDao.java

```diff
@@ -85,6 +87,17 @@
         return ImmutableList.copyOf(Arrays.asList(searchHits));
     }
 
+    // returns no of documents in an index
+    @Override 
+    public long getDocCount (String index) throws IOException {
```

Comment:
Optional:

getDocumentCount

In variable names, feel free to use the complete words.